### PR TITLE
Avoid circular dependency between superset config and superset package

### PR DESCRIPTION
**Pull request by bkyryliuk**
### CATEGORY

Choose one

- [X] Bug Fix

### SUMMARY

Small refactor to avoid circular dependency between superset config and superset package.
Current code doesn't support adding custom security manager as importing SupersetSecurityManager calls __init__.py that initializes the config.
### TEST PLAN
* local testing + testing in DBX environment

### CODE REVIEW

**Comment by dpgaspar** [comment]
I would rename `superset/security/security.py` to `superset/security/manager.py`.
You need to add the Apache license to `superset/security/__init__.py`

**Comment by bkyryliuk** [comment]
@dpgaspar did you mean import `SupersetSecurityManager ` in `superset/security/__init__.py`

**Comment by dpgaspar** [comment]
Yep, just edited my previous reply

**Comment by bkyryliuk** [comment]
@dpgaspar done

File: superset/config.py [hunk 1]
```diff
@@ -61,6 +61,9 @@ def _try_json_readfile(filepath):
         return None
 
 
+STATS_LOGGER = StatsdStatsLogger
```
Comment (dpgaspar) [inline superset/config.py]: This is not needed right?

Comment (bkyryliuk) [reply superset/config.py]: ooops, it's not

**Review by mistercrunch — APPROVED** [review]

**Review by dpgaspar — APPROVED** [review]

**Pull request merged**